packages/angular/.eslintrc.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  root: true,
+  overrides: [
+    {
+      files: ['**/*.ts'],
+      parserOptions: {
+        project: ['tsconfig.json'],
+        tsconfigRootDir: __dirname,
+        createDefaultProgram: true,
+      },
+      extends: [
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@angular-eslint/recommended',
+        // This is required if you use inline templates in Components
+        'plugin:@angular-eslint/template/process-inline-templates',
+        'amplify-ui',
+      ],
+      rules: {
+        /*
+         * TODO: turn the following settings on after fixing the
+         * corresponding linting issues.
+         */
+        '@typescript-eslint/explicit-module-boundary-types': 'warn',
+        '@typescript-eslint/member-ordering': 'warn',
+        '@typescript-eslint/no-empty-function': 'warn',
+        '@typescript-eslint/no-explicit-any': 'warn',
+        '@typescript-eslint/no-inferrable-types': 'warn',
+        '@typescript-eslint/no-unnecessary-type-assertion': 'warn',
+        '@typescript-eslint/no-unsafe-argument': 'warn',
+        '@typescript-eslint/no-unsafe-assignment': 'warn',
+        '@typescript-eslint/no-unsafe-member-access': 'warn',
+        'prefer-const': 'warn',
+        'prefer-destructuring': 'warn',
```

Comment:
I temporarily put these to `warn` so that feature branch can keep track of which errors to fix without breaking linting. 

I will remove these lines (ie. put these back to `'error'`) as I fix the errors.